### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by sylvaingaudan**
- new structure for the documentation
- update of the documentation
- add schemagen in the documentation

### CODE REVIEW

File: pesto-cli/docs/package_docker_image.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+!!! Success "xxx-service:1.0.0.dev0-gpu"
+
+
+!!!note "Nvidia drivers"
+    If you have a computer with nvidia drivers & a gpu you can try to run
+    ```bash
+    pesto build {PESTO_PROJECT_ROOT} -p gpu
+    pesto test {PESTO_PROJECT_ROOT} -p gpu --nvidia
+    ```
+    It which should do the same as above but with gpu support (and actually run the process on gpu).
```
Comment (MatthieuLamothe) [inline pesto-cli/docs/package_docker_image.md:68]: I think there is a typo: `It which should` -> `It should`

File: pesto-cli/docs/deploy_run_docker_image.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+```bash
+pesto run docker '{"image":"file:///opt/algo-service/pesto/tests/resources/test_1/input/image.png"}' algo-service:1.0.0.dev0 /tmp/output_pesto.json
+```
+
+- ## pesto run local
+
+With this option, you can run the algorithm from a python environment where all dependencies are installed. 
+
+Either from your local environment:
```
Comment (MatthieuLamothe) [inline pesto-cli/docs/deploy_run_docker_image.md:28]: Only present the `run local` from docker environment option ?

File: pesto-cli/docs/pesto_list.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# pesto list
```
Comment (MatthieuLamothe) [inline pesto-cli/docs/pesto_list.md:1]: To complete or delete

**Comment by sylvaingaudan** [comment]
Cancel pull request (MR goes to branch doc-update)

**Pull request closed without merging**